### ISSUE TITLE
Haven wallet forgets node selection

### ISSUE THREAD

**Issue by nahuhh** [cake-tech/cake_wallet#334]
When you add a custom node to your Haven wallet, you are able to connect without issue.
If you close the app and relaunch, it connects to the _default Haven node_. You can still manually connect to the custom node, but on each launch it will be back set to Haven's.

Youd assume that deleting Haven's node and only leaving the custom may work, but instead the wallet will launch with no node selected, and you have to manually select.

Tldr
Haven node selection doesnt remember the last node used

V4.4.0

**Comment by nahuhh** [comment]
Just wanted to update.

It doesn't appear to effect iOS.
Issue does not effect monero nodes.
I have not checked litecoin or bitcoin nodes.
Still a problem in 4.4.1

**Comment by nahuhh** [comment]
Appears to be fixed with https://github.com/cake-tech/cake_wallet/pull/388

Waiting for release (or beta) to confirm

**Comment by nahuhh** [comment]
Confirmed fixed
